### PR TITLE
fix(images): add image build status to system log and show all builded images

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: src/api/image-builder.ts [hunk 1]
```diff
@@ -79,24 +92,28 @@ export function getImages(): Promise<any> {
...
-              name: image.RepoTags[0].split(':')[0],
-              version: image.RepoTags[0].split(':')[1],
-              created: format(new Date(image.Created * 1000), 'DD.MM.YYYY HH:mm:ss'),
-              createdAgo: distanceInWordsToNow(new Date(image.Created * 1000)),
-              size: getHumanSize(image.Size),
-              dockerfile: null,
-              initsh: null
-            };
-          });
+          }).filter(d => d ? d : false);
```
Comment (jkuri) [inline src/api/image-builder.ts]: `.filter(Boolean);` is nicer and does the same thing. What do you think?

Comment (Izak88) [reply src/api/image-builder.ts]: did not know about that, awesome! :). I'll update that

File: src/api/image-builder.ts [hunk 2]
```diff
@@ -40,12 +40,25 @@ export function buildDockerImage(data: ImageData): void {
         });
         output.on('finish', () => {
           msg.message = `image ${data.name} build successfully completed`;
```
Comment (jkuri) [inline src/api/image-builder.ts]: please use [type](https://github.com/bleenco/abstruse/blob/master/src/api/logger.ts#L5-L9) that we already have defined and don't apply every property for itself. i.e.
```ts
import { LogMessageType } from './logger';
...
const msg: LogMessageType = {
  message: `....`,
  type: 'error',
  notify: true
};
logger.next(msg);
```

File: src/api/image-builder.ts (hunk 1 above)
Comment (Izak88) [reply src/api/image-builder.ts]: done

File: src/api/image-builder.ts (hunk 2 above)
Comment (Izak88) [reply src/api/image-builder.ts]: done

**Pull request merged**